tidy3d/plugins/mode/mode_solver.py

```diff
@@ -1141,6 +1143,324 @@ def plot_field(
             **sel_kwargs,
         )
 
+    def plot(
+        self,
+        ax: Ax = None,
+        **patch_kwargs,
+    ) -> Ax:
+        """Plot each mode plane simulation's components.
```

Comment:
"each mode plane"? is this correct because it seems to only plot the single mode plane?

Reply:
Thanks, it's a typo. I will remove it.